Documentation/Input/InputEvents.md

```diff
@@ -1,23 +1,169 @@
 # Input Events
 
-At the script level you consume input events by implementing one of the event handler interfaces shown in the table below.
+The list below outlines all available input event interfaces to be implemented by a custom MonoBehavior component. These interfaces will be called by the MRTK Input System to handle custom app logic based on user input interactions. [Pointer input events](pointers.md#pointer-event-interfaces) are handled slightly differently than the standard input event types below.
```

Comment:
I feel like the old stuff had an interesting tidbit about only recieving events when in focus - it feels like that's still an important part to capture.

We may not need to add the part about why the old global thing sucks but the new thing is awesome, but the part that talked about how to expect to get them seems to be useful to keep around.

Reply:
I don't understand the comment. There is an important tag saying you only receive events when in focus

Reply:
The important tag shows up later, after the reader has already gone through the interface information (and potentially already started trying to implement those interfaces). Moreover, the important comment shows up in the section called "registering for global events"

Basically, I think that still having the important tag stuff up here about requiring focus is still good to tell people before telling them about the interfaces (i.e. 

step 1) tell people how to start listening for events in general
step 2) tell people exactly which interfaces for which events

Reply:
ok I see. @wiwei do you think at the very top or with the "Input events in action" since that actually explains how to utilize them?